.pre-commit-config.yaml

```diff
@@ -88,13 +88,13 @@ repos:
       # pyenv and/or virtualenv activated; it may not have been e.g. if
       # committing from a GUI tool that was not launched from an activated
       # shell.
-      - id: mypy
-        name: mypy
-        entry: script/run-in-env.sh mypy
-        language: script
-        types: [python]
-        require_serial: true
-        files: ^homeassistant/.+\.py$
+      # - id: mypy
+      #  name: mypy
+      #  entry: script/run-in-env.sh mypy
+      #  language: script
+      #  types: [python]
+      #  require_serial: true
+      #  files: ^homeassistant/.+\.py$
```

Comment:
This change doesn't seem to be related (and not wished for either 😬 )

Reply:
Oh shit, sorry for that :open_mouth:.
I will try to rebase my branch.

Reply:
Oh shit, sorry for that :open_mouth:.
I will try to rebase my branch.

Reply:
Rebase is done.
This is why I commented out the mypy hook:
```
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

homeassistant/components/moehlenhoff_alpha2/__init__.py:52: error: "dict" is not subscriptable  [misc]
homeassistant/components/moehlenhoff_alpha2/__init__.py:52: error: "dict" is not subscriptable, use "typing.Dict" instead  [misc]
homeassistant/components/moehlenhoff_alpha2/__init__.py:65: error: "dict" is not subscriptable, use "typing.Dict" instead  [misc]
Found 3 errors in 1 file (checked 1 source file)
```
If I use `typing.Dict` instead, pylint complains about it.

Reply:
Add at the top of the module:
```py
from __future__ import annotations
```

Reply:
Thanks!
One mypy error persists:
```
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

homeassistant/components/moehlenhoff_alpha2/__init__.py:54: error: "dict" is not subscriptable  [misc]
Found 1 error in 1 file (checked 1 source file)
```

Reply:
That should work if you're running Python 3.9+.

Reply:
I am using the dev container.

Reply:
Never mind, the issue went away somehow :thinking: